modules/channel-vonage/src/backend/client.ts

```diff
@@ -383,7 +271,15 @@ export async function setupMiddleware(bp: typeof sdk, clients: Clients) {
     order: 100
   })
 
-  async function outgoingHandler(event: sdk.IO.Event, next: sdk.IO.MiddlewareNextCallback) {
+  bp.events.registerMiddleware({
+    description: 'Translate index responses to their corresponding payloads',
+    direction: 'incoming',
+    handler: incomingHandler,
+    name: 'vonage.indexResponse',
+    order: 0
```

Comment:
Maybe add a comment saying that this value should be higher than any other middlewares that might mutate/alter the payload content of an event?

Reply:
So I did a quick search and it seems most of our incoming middleware have a "end of queue" priority (such as 100, 120, etc).

The only ones that had low priority are the hitl modules. The OG hitl module has an incoming middleware of priority 2, and the hitlnext module has an incoming middleware priority of 0. So it's possible we might have some trouble with the hitlnext module. If we find that we have some trouble with index reponses on that module, we could make the order 2 for this module as well so it passes after our middleware